modules/programs/fzf.nix

```
@@ -88,6 +88,23 @@ in {
       '';
     };
 
+    tmux = {
+      enableShellIntegration = mkEnableOption ''
+        setting <literal>FZF_TMUX=1</literal> which causes shell integration to use fzf-tmux
+      '';
+
+      shellIntegrationOptions = mkOption {
+        type = types.listOf types.str;
+        default = [ ];
+        example = literalExample ''[ "-d 40%" ]'';
+        description = ''
+          If <options>programs.fzf.tmux.enableShellIntegration<options> is set to true,
```

The build is failing due to the end tag being incorrect. Also, I think it should be `option` not `options`.

```suggestion
          If <option>programs.fzf.tmux.enableShellIntegration</option> is set to true,
```